erpnext/setup/doctype/party_type/party_type.py

```
@@ -26,13 +26,24 @@ class PartyType(Document):
 @frappe.validate_and_sanitize_search_inputs
 def get_party_type(doctype, txt, searchfield, start, page_len, filters):
 	cond = ""
+	account_type = None
 	if filters and filters.get("account"):
 		account_type = frappe.db.get_value("Account", filters.get("account"), "account_type")
 		cond = "and account_type = '%s'" % account_type
```

Ruff (0.12.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)